rules/framework/vfs_overlay.bzl

```diff
@@ -33,9 +33,9 @@ def _get_vfs_parent(ctx):
 # Make roots for a given framework. For now this is done in starlark for speed
 # and incrementality. For imported frameworks, there is additional search paths
 # enabled
-def _make_root(vfs_parent, bin_dir_path, build_file_path, framework_name, root_dir, extra_search_paths, module_map, hdrs, private_hdrs, has_swift):
-    extra_roots = []
+def _make_root(vfs_parent, bin_dir_path, build_file_path, framework_name, swiftmodule_contents, root_dir, extra_search_paths, module_map, hdrs, private_hdrs, has_swift):
```

Comment:
added `swiftmodule_contents` here... maybe we should break this into multiple lines for readability